Protobuild.toml

```diff
@@ -0,0 +1,33 @@
+version = "unstable"
+generator = "gogoswarm"
+plugins = ["grpc", "deepcopy", "storeobject", "raftproxy", "authenticatedwrapper"]
+
+# Control protoc include paths. Below are usually some good defaults, but feel
+# free to try it without them if it works for your project.
+[includes]
+  # Include paths that will be added before all others. Typically, you want to
+  # treat the root of the project as an include, but this may not be necessary.
+  before = [".", "./protobuf"]
```

Comment:
Include `./protobuf` will let the proto files in `swarmkit/api` can import `swarmkit/protobuf/plugin/plugin.proto` using relative path `plugin/plugin.proto`. 

If not include `./protobuf`, we must change `plugin/plugin.proto` imports to full path. It's really not necessary but a choice. We can discuss which is better.